tests/conftest.py

```diff
@@ -486,7 +487,7 @@ def aperture_scatterguard(RE):
         patch_async_motor(ap_sg.scatterguard.x),
         patch_async_motor(ap_sg.scatterguard.y),
     ):
-        RE(bps.abs_set(ap_sg, ap_sg.aperture_positions.SMALL))  # type:
+        RE(bps.abs_set(ap_sg, ap_sg.aperture_positions.SMALL))  # type: ignore
```

Comment:
Nit:  A better fix would be to add `assert ap_sg.aperture_positions is not None` before this